### PR TITLE
update go version to 1.17.3

### PR DESCRIPTION
**Pull request by bosorawis00**
Upgrading go version to resolve CVE-2021-39293 (`archive/zip` vulnerability) 
https://access.redhat.com/security/cve/cve-2021-39293

### CODE REVIEW

File: .go-version [hunk 1]
```diff
@@ -1 +1 @@
-1.16.5
+1.16.8
```
Comment (ashutosh-narkar) [inline .go-version:1]: Let's use `1.17` then. WDYT ?

Comment (bosorawis00) [reply .go-version:1]: Hi!, we'd have to go to `1.17.1` though since same vulnerability exists in `1.17`. But that works as well

**Comment by ashutosh-narkar** [comment]
`1.17.1` sounds fine. Can you rebase and we can merge this!

**Comment by ashutosh-narkar** [comment]
@bosorawis00 can you please fix the commit message ?

**Comment by srenatus** [comment]
Thanks for raising this and contributing a fix. @ashutosh-narkar I wonder we shouldn't go to 1.17.3 instead, the latest release of the 1.17 series?

**Comment by bosorawis00** [comment]
@ashutosh-narkar yup! sorry about that. I also bump to 1.17.3 per @srenatus suggestion

**Review by ashutosh-narkar — APPROVED** [review]
Thanks @bosorawis00 !

**Pull request merged**